crates/redstone/src/types/signer_address.rs

```diff
@@ -30,3 +32,32 @@ impl From<Vec<u8>> for SignerAddress {
         Self::new(buff)
     }
 }
+
+impl fmt::Display for SignerAddress {
```

Comment:
Similar comment, but unlike in the FeedId they are not implemented directly on type SignerAddress tho